cmd/client/main.go

```diff
@@ -24,15 +24,17 @@ func main() {
 	// читаем строку из консоли
 	long, err := reader.ReadString('\n')
 	if err != nil {
-		log.Fatalf(err.Error())
+		var msg = err.Error()
+		log.Fatalf("error %s", msg)
```

Comment:
Тут можно обойтись без лишних операций:
```suggestion
		log.Fatalf("error %v", err)
```